internal/arr/model/radarr.go

```diff
@@ -16,3 +16,9 @@ type Movie []struct {
 	} `json:"movieFile"`
 	QualityProfileID int `json:"qualityProfileId"`
 }
+
+type TagObjects []struct {
```

Comment:
This is specific to radarr, let's call this TagMovies

Reply:
Updated.